lib/inputs/file-pattern/static.ts

```diff
@@ -48,14 +48,15 @@ export function getMatchingFiles(
       if (actionName !== "find-files-by-pattern") {
         continue;
       }
-      if (!(typeof extractedLayers[filePath][actionName] === "string")) {
-        throw new Error("expected string");
+
+      if (!Buffer.isBuffer(extractedLayers[filePath][actionName])) {
+        throw new Error("expected a buffer");
```

Comment:
please continue and not throw :) not sure if this check is needed at all

Reply:
Could you please elaborate why the behaviour should be changed in line with this PR. Why now continue and not throw - which of my changes needs this change in behaviour?

Reply:
you're correct 👍 I thought you added the throw